### PR TITLE
vCore: logging operator test refactoring, switch from the elasticsearch to the loki + internal odf fix

### PR DESCRIPTION
**Pull request by elenagerman**
vCore: logging operator test refactoring, switch from the elasticsearch to the loki + internal odf fix

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -30,18 +30,20 @@ require (
 	golang.org/x/exp v0.0.0-20240604190554-fc45aab8b7f8
 	gopkg.in/k8snetworkplumbingwg/multus-cni.v4 v4.0.2
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.29.5
-	k8s.io/apimachinery v0.29.5
+	k8s.io/api v0.30.2
```
Comment (kononovn) [inline go.mod:33]: We can't switch to v0.30.2

File: go.mod [hunk 2]
```diff
@@ -30,18 +30,20 @@ require (
...
+	k8s.io/apimachinery v0.30.2
```
Comment (kononovn) [inline go.mod:34]: We can't switch to v0.30.2 . It'll cause deps conflicts

**Review by kononovn — CHANGES_REQUESTED** [review]
Please fix deps issue

File: go.mod (hunk 1 above)
Comment (elenagerman) [reply go.mod:33]: sure, I remember you told me. not finished yet with this one, sorry

Comment (elenagerman) [reply go.mod:33]: pushed to see all changes in a normal way

**Review by kononovn — CHANGES_REQUESTED** [review]
go.mod/go.sum lgtm

**Pull request merged**